src/client.cpp

```diff
@@ -358,6 +361,12 @@ GAME_ERROR RunFrame(void)
   if (!CLIENT)
     return GAME_ERROR_FAILED;
 
+  // Trigger the frame time callback before running the core.
+  uint64_t current = timer.microseconds();
+  int64_t delta = current - FRAME_TIME_LAST;
+  FRAME_TIME_LAST = current;
+  CLIENT_BRIDGE->FrameTime(delta);
```

Comment:
delta will be a large value on the first call to RunFrame()

Reply:
Mmm, we should likely attribute for that on initialization.

Reply:
Set it to timer.microseconds() over at https://github.com/kodi-game/game.libretro/pull/49/files#diff-34f48ad91ac6c202ac60b0348ae90e30R111